railties/test/generators/shared_generator_tests.rb

```diff
@@ -203,7 +203,8 @@ def test_generator_for_active_storage
 
     unless generator_class.name == "Rails::Generators::PluginGenerator"
       assert_file "#{application_path}/app/javascript/packs/application.js" do |content|
-        assert_match(/^require\("@rails\/activestorage"\)\.start\(\)/, content)
+        assert_match(/^import \* as ActiveStorage from "@rails\/activestorage"/, content)
+        assert_match(/^ActiveStorage.start\(\)/, content)
       end
     end
```

Comment:
There's one more test down below to update: https://github.com/rails/rails/blob/3cd1b19ff2e49642191c598d620b87ffe5ec6cda/railties/test/generators/shared_generator_tests.rb#L266-L268